kubernetes/zenko/charts/cloudserver/values.yaml

```diff
@@ -64,6 +64,12 @@ logging:
 
 allowHealthchecksFrom: '0.0.0.0/0'
 
+health:
+  port: http
```

Comment:
I think just having the `initialDelaySeconds` configurable would suffice. Typically we don't want user configurable values that could break the deployment and being able to change the container port (which isn't configurable either). Same pretty much goes for the health check route because we want to ensure that the health route we want is always used.

I know the backbeat chart does it this way however this is will likely be removed soon as we are refactoring the backbeat chart.